string SERV_SEARCH = search
string SERV_GOHOME = goHome

string TOPIC_TURTLESTATE = turtle_state
string TOPIC_TURTLEREQUEST = turtle_request
string TOPIC_TURTLERESPONSE = turtle_response

string TOPIC_DRONESTATE = drone_state
string TOPIC_DRONEREQUEST = drone_request
string TOPIC_DRONERESPONSE = drone_response

string NODE_TURTLEHUB = turtle_hub

string STATE_STATIONARY = Stationary
string STATE_MOVING = Moving